kong/templates/kong_defaults.lua

```diff
@@ -208,6 +208,7 @@ tracing_sampling_rate = 0.01
 wasm = off
 wasm_filters_path = NONE
 wasm_dynamic_module = NONE
+wasm_filters = NONE
```

Comment:
can we make `bundled,user` be the default? since `wasm` is `off` by default that's harmless, and it would mean it would only take one setting (`wasm = on`) to be able to use Wasm with the bundled filter, and the setup instructions from previous Kong versions would still work for enabling user filters ("set `wasm = on` and `wasm_filters_path`")

Reply:
WFM, for some reason I thought `plugins` was `off` by default and wanted to match it, but I just checked and see that it is also `bundled`.